src/main/java/duke/Task.java

```diff
@@ -0,0 +1,41 @@
+package duke;
+
+import java.io.Serializable;
+public abstract class Task implements Serializable {
```

Comment:
Nice use of Java's `Serializable` interface to read/write the tasks to a file!

I wanted to test your implementation of this, but unfortunately the issue caused by the gitignore'd images mentioned above meant that I couldn't.